LICENSE-APACHEv2

```diff
@@ -186,7 +192,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Status Research & Development GmbH
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
...

Reply:
The original is actually wrong; read the text just before it.  Third parties are not supposed to add "Copyright 2018 Status Research" to their own files...